mycpp/my_runtime.cc

```diff
@@ -232,7 +232,7 @@ Str* Str::strip() {
 
 // Used for CommandSub in osh/cmd_exec.py
 Str* Str::rstrip(Str* chars) {
-  assert(0);
+  NotImplemented(); // Uncalled
```

Comment:
It is used, except the comment is now out of date.  The usage is in core/executor.py.

e.g. `grep rstrip */*.py`